### PR TITLE
OpenAPI spec changes to enable card enrolment 

### PR DESCRIPTION
**Pull request by k-irishina**
Main PR: https://github.com/techcloud0/bankaxept-epayment-platform/pull/1672

### CODE REVIEW

File: openapi/integrator/wallet/partner.yaml [hunk 1]
```diff
@@ -58,7 +58,30 @@ paths:
           $ref: '../components.yaml#/components/responses/500Error'
         '503':
           $ref: '../components.yaml#/components/responses/503Error'
-  /payment-tokens/suspensions:
+
+  /wallet/payment-tokens/authentication/complete:
+    description: When the authentication process has been completed, the user agent is redirected to this endpoint.
```
Comment (k-irishina) [inline openapi/integrator/wallet/partner.yaml:63]: Feel free to suggest a better description!

File: openapi/integrator/wallet/bankaxept.yaml [hunk 2]
```diff
@@ -127,6 +127,7 @@ components:
         - merchantAggregatorId
         - nin
         - accountNumber
+        - merchantReference
```
Comment (k-irishina) [inline openapi/integrator/wallet/bankaxept.yaml:130]: else the merchant won't be able to correlate.

File: openapi/integrator/wallet/partner.yaml [hunk 3]
```diff
@@ -251,6 +297,11 @@ components:
         - Failed
         - AuthenticationFailed
 
+    AuthenticationStatus:
+      type: string
+      example: Accepted
+      enum: [ Accepted, Failed ]
```
Comment (thxmasj) [inline openapi/integrator/wallet/partner.yaml:303]: Didn't know this syntax worked. We've used lists with dashes before.

**Review by thxmasj — APPROVED** [review]
Wait with merge till platform PR is ready.

**Pull request merged**